safe_transaction_service/history/services/collectibles_service.py

```diff
@@ -363,7 +399,47 @@ def get_collectibles_with_metadata(
                     metadata,
                 )
             )
-        return collectibles_with_metadata
+        return collectibles_with_metadata, count
+
+    def get_collectibles_with_metadata(
+        self,
+        safe_address: ChecksumAddress,
+        only_trusted: bool = False,
+        exclude_spam: bool = False,
+    ) -> List[CollectibleWithMetadata]:
+        """
+         Get collectibles v1 returns no paginated response
+        :param safe_address:
+        :param only_trusted: If True, return balance only for trusted tokens
+        :param exclude_spam: If True, exclude spam tokens
+        :return: collectibles
+        """
+        collectibles, _ = self._get_collectibles_with_metadata(
+            safe_address, only_trusted, exclude_spam
+        )
+        return collectibles
+
+    def get_collectibles_with_metadata_paginated(
+        self,
+        safe_address: ChecksumAddress,
+        only_trusted: bool = False,
+        exclude_spam: bool = False,
+        limit: int = 0,
```

Comment:
IMHO limit should have a reasonable default, like `limit: int = 10`. `0` does not makes sense to me in this case